scripts/quality/check_codacy_zero.py

```
@@ -16,84 +16,274 @@
+
+    for candidate in candidates:
+        try:
+            open_issues = _scan_candidate(
+                candidate=candidate,
+                owner=owner,
+                repo=repo,
+                token=token,
+                branch=branch,
+                findings=findings,
+            )
+            return open_issues, findings
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)